src/main/webapp/index.html

```diff
@@ -1,12 +1,89 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
-    <title>Nina's Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8" />
+    <title>School Form</title>
+    <link rel="stylesheet" href="style.css" />
     <script src="script.js"></script>
   </head>
   <body>
-      <p> Hello </p>
+    <p>Rate your classes and professors!</p>
+    <form action="/data" method="POST">
+      <p>Rate your class:</p>
+      <textarea name="class-input">Write opinion on class here.</textarea>
+      <br /><br />
+
+      <p>How would you rate this class?</p>
+      <input
+        type="radio"
+        id="excellent"
+        name="rating-class"
+        value="excellent"
+      />
+      <label for="excellent">Excellent.</label>
+      <input
+        type="radio"
+        id="acceptable"
+        name="rating-class"
+        value="acceptable"
```

Comment:
I'm not sure if it's good or bad style to have all of these things on separate lines? since all together this line wouldn't be over 80 characters, maybe put the entire input tag on the same line? but this might just be a matter of personal preference haha

Reply:
Yess I also wanted to do that but for some reason prettier is being weird :(

Reply:
^^ prettier just did that to me too! weird...

Reply:
rip, if it persists maybe it could be an actual buug

Reply:
Hm, that's pretty weird if it's under 80 lines and still insisting on wrapping...

Reply:
Yeah, we are not sure what's happening with prettier hahaha. It's definitely not normal right?